crates/matrix-sdk/examples/get_profiles.rs

```diff
@@ -7,6 +7,7 @@ use matrix_sdk::{
 use url::Url;
 
 #[derive(Debug)]
+#[allow(dead_code)]
```

Comment:
Why is this dead if it's clearly used?

Reply:
It's written, but never read except for through `Debug`, which Nightly rustc no longer considers as a "real" use.